$NetBSD: patch-configure,v 1.8 2021/02/01 20:51:24 gdt Exp $

HAVE_BSD_STRUCT_IP_MREQ_HACK does not work on SunOS
Results in: can't setsockopt IP_MULTICAST_IF(...): No route to host

--- configure.orig	2017-10-03 14:59:10.000000000 +0000
+++ configure
@@ -20631,7 +20631,7 @@ cat confdefs.h - <<_ACEOF >conftest.$ac_
 int
 main ()
 {
-#if (defined(__FreeBSD__) && ((__FreeBSD_version >= 500022 && __FreeBSD_version < 700000) || (__FreeBSD_version < 500000 && __FreeBSD_version >= 440000))) || (defined(__NetBSD__) && defined(__NetBSD_Version__) && __NetBSD_Version__ >= 106010000) || defined(__OpenBSD__) || defined(__APPLE__) || defined(__DragonFly__) || defined(__sun)
+#if (defined(__FreeBSD__) && ((__FreeBSD_version >= 500022 && __FreeBSD_version < 700000) || (__FreeBSD_version < 500000 && __FreeBSD_version >= 440000))) || (defined(__NetBSD__) && defined(__NetBSD_Version__) && __NetBSD_Version__ >= 106010000) || defined(__OpenBSD__) || defined(__APPLE__) || defined(__DragonFly__)
   return (0);
 #else
   #error No support for BSD struct ip_mreq hack detected
